api/Controllers/AuthController.cs

```diff
@@ -15,40 +18,82 @@ namespace iTechArt.Hotels.Api.Controllers
     public class AuthController : Controller
     {
         private readonly IOptions<AuthOptions> _authOptions;
+        private readonly IHashPasswords _hashPasswordsService;
         private readonly HotelsDatabaseContext _hotelsDb;
 
-        public AuthController(IOptions<AuthOptions> authOptions, HotelsDatabaseContext hotelsDb)
+        public AuthController(IOptions<AuthOptions> authOptions, HotelsDatabaseContext hotelsDb, IHashPasswords hashPasswordsService)
```

Comment:
Try to fit within 120 characters on one line.

You've already faced the same challenge with constructor arguments in Angular :)